homeassistant/components/reolink/icons.json

```diff
@@ -200,6 +200,15 @@
       "ptz_pan_position": {
         "default": "mdi:pan"
       },
+      "battery_percent": {
+        "default": "mdi:battery"
+      },
```

Comment:
IIRC this can be omitted as the sensor has the battery as its device_class and thus falls back to the icon defined by the sensor component.

Reply:
Thanks for pointing this out!